pkg/controller/controller.go

```diff
@@ -334,6 +343,36 @@ func ttls() []string {
 	return []string{"15m", "30m", "1h", "2h", "4h"}
 }
 
-func categories() []string {
-	return []string{"explanation"}
+func (c *Controller) categories() []string {
+	validators := c.p.Validators()
+	categories := make([]string, 0, len(validators))
+	for v := range validators {
+		categories = append(categories, v)
+	}
+	categories = append(categories, "explanation")
```

Comment:
Such an outlier. I want to make explanation also a validator https://github.com/abcxyz/jvs/issues/302

Reply:
ack